README.md

```diff
@@ -178,6 +178,12 @@ the Design of Network-based Software Architectures](https://www.ics.uci.edu/~fie
 * [Vertx-Web](https://github.com/vert-x3/vertx-web) - Vert.x-Web is a set of building blocks for building web applications with Vert.x, a toolkit for building reactive applications on the JVM.
 * [Dropwizard](https://github.com/dropwizard/dropwizard) - A framework for developing ops-friendly, high-performance, RESTful web services.
 
+### Scala
+
+* [Chaos](https://github.com/mesosphere/chaos) - A lightweight framework for writing REST services in Scala.
+* [Play](https://www.playframework.com) - The High Velocity Web Framework For Scala and Java.
```

Comment:
From what I understand, Play is a general purpose MVC framework and not a REST API framework. It doesn't make sense to mention Play here.

Reply:
No problem @fzaninotto